src/modules/fw_pos_control/FixedwingPositionControl.cpp

```diff
@@ -1172,7 +1168,7 @@ FixedwingPositionControl::control_auto_velocity(const float control_interval, co
 	_att_sp.yaw_body = _yaw;
 
 	tecs_update_pitch_throttle(control_interval,
-				   position_sp_alt,
+				   NAN,
```

Comment:
Is this NAN required? I think the currently interface to TECS just discards the altitude setpoint if the height rate setpoint is finite.

Reply:
i believe you had the most recent look at this @sfuhrer ;) -- but im not opposed to be more explicit if we can on intentions with the interface (i.e. explicitly saying we dont have a height setpoint). @Jaeyoung-Lim i suppose with this nan it worked as you expected?

Reply:
I think we're fine setting it to NAN, as long as then height rate is set to a finite value. If both are NAN it looks to me like we get an issue. 
https://github.com/PX4/PX4-Autopilot/blob/4e4ba402890610852d1cde7fef77cd6e49e838b6/src/lib/tecs/TECS.cpp#L168

Reply:
>  If both are NAN it looks to me like we get an issue.

this is a problem then. inputs are not properly handled.